internal/stack/stacks.go

```diff
@@ -127,12 +133,15 @@ func getStackBuffer(all bool) []byte {
 	}
 }
 
-func parseFirstFunc(line string) string {
+func parseFunc(line string) string {
 	line = strings.TrimSpace(line)
 	if idx := strings.LastIndex(line, "("); idx > 0 {
 		return line[:idx]
 	}
-	panic(fmt.Sprintf("function calls missing parents: %q", line))
+	if idx := strings.LastIndex(line, "created by"); idx >= 0 {
+		return strings.TrimPrefix(line, "created by ")
+	}
```

Comment:
since it's not part of the stack, may be better to skip parsing these till it's required